sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -616,15 +624,16 @@ static status_t personalize_endorse_certificates(ujson_t *uj) {
       // We just prepared the set of wrapped certificates, let's trust that the
       // data is correct and does not need more validation.
       memcpy(&crth, certs, sizeof(crth));
-      certs += sizeof(crth);
       PERSO_TLV_GET_FIELD(Crth, NameSize, crth, &name_len);
-      memcpy(name, certs, name_len);
+      memcpy(name, (void *)(certs + sizeof(crth)), name_len);
```

Comment:
```suggestion
      memcpy(name, certs + sizeof(crth), name_len);
```
I don't think you need the cast here do you?